src/Infrastructure/Persistence/Doctrine/Migrations/Version20230215135519.php

```diff
@@ -11,7 +11,7 @@ final class Version20230215135519 extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return 'Swap FK in RegulationOrder <-> RegulationCondition relationship';
+        return 'Swap FK in RegulationOrder <-> Condition relationship';
```

Comment:
C'est pour l'historique

```suggestion
        return 'Swap FK in RegulationOrder <-> RegulationCondition relationship';
```